statement-summary-tables.md

```diff
@@ -93,6 +209,8 @@ Basic fields:
 - `SAMPLE_USER`: The users who execute SQL statements of this category. Only one user is taken.
 - `PLAN_DIGEST`: The digest of the execution plan.
 - `PLAN`: The original execution plan. If there are multiple statements, the plan of only one statement is taken.
+- `PLAN_CACHE_HITS`: The total number of times that SQL statements of this category hit the plan cache.
+- `PLAN_IN_CACHE`: Determines whether the previous execution of SQL statements of this category hit the plan cache.
```

Comment:
Would `indicates` be better? `determines` may be regarded as `decides`.

Reply:
updated